### PR TITLE
Fix 2 minor README typos

### PR DESCRIPTION
**Pull request by thomassdk**
-------

## Description

Fix some README typos

### CODE REVIEW

**Review by ArmandoGraterol — APPROVED** [review]
Congrats on your first PR on this side of the pond :)

File: Readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Colony CDapp
 
-A interation of the Colony Dapp sporting both a fully decentralized operating mode, as well as a mode enhanced by a metadata caching layer
+A iteration of the Colony Dapp sporting both a fully decentralized operating mode, as well as a mode enhanced by a metadata caching layer
```
Comment (willm30) [inline Readme.md:3]: If we're being precise, this should read `an iteration`

File: Readme.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ npm run dev
 
 This will build your local docker images, then attempt to start them _(the local dev environment runs inside a couple of docker containers).
 
-On the next start, assuming no key cache files changed, it will skip the image building step _(as it will just used the cached version)_, and go strait to starting your environment
+On the next start, assuming no key cache files changed, it will skip the image building step _(as it will just used the cached version)_, and go straight to starting your environment
```
Comment (willm30) [inline Readme.md:38]: Full stop at the end of the paragraph

**Review by willm30 — APPROVED** [review]
LGTM

**Pull request merged**